src/map/skill.cpp

```diff
@@ -5582,7 +5582,7 @@ int32 skill_castend_damage_id (struct block_list* src, struct block_list *bl, ui
 
 	case RG_BACKSTAP:
 		{
-			if (!check_distance_bl(src, bl, 0)) {
+			if (distance_bl(src, bl) <= 1) {
```

Comment:
Shouldn't this still be check_distance_bl just with 1 as range?

Reply:
When using Flying Kick the distance is 0
When using Backstab the distance is 1
That's why I used distance_bl instead.

Reply:
I don't get the explanation. Check distance is already defined like this:
```
	if (dx < 0) dx = -dx;
	if (dy < 0) dy = -dy;
	return ((dx<dy?dy:dx) <= distance);
```